### ISSUE TITLE
Failure to create indexer DB client 

### ISSUE THREAD

**Issue by andygolay** [movementlabsxyz/movement#843]
**Describe the bug**
In the bridge process compose log, the following warning appears:
```
WARN bridge_service: Failed to create indexer db client:
Failed to connect to postgresql instance: missing "=" after ""postgresql://
postgres:password@localhost:5432"" in connection info string
```

This results in events and actions not being indexed:

```
WARN bridge_service: No indexer db client found. Event not indexed
WARN bridge_service: No indexer db client found. Action not indexed
```

**To Reproduce**
```
CELESTIA_LOG_LEVEL=FATAL CARGO_PROFILE=release CARGO_PROFILE_FLAGS=--release  nix develop --extra-experimental-features nix-command --extra-experimental-features flakes --command bash  -c "just bridge native build.setup.eth-local.celestia-local.bridge.bridge-indexer --keep-tui"
```

**Expected behavior**
- Indexer db client should be found
- Events and actions should be indexed

**Comment by 0xmovses** [comment]
https://github.com/movementlabsxyz/movement/pull/824 will merge this

**Comment by l-monninger** [comment]
The above logs are valid behavior, but you should not include quotes around your connection string. Both were already addressed in #821 .

**Comment by l-monninger** [comment]
Confirmed, closing.
